mpt/core/src/main/java/org/apache/james/mpt/script/SimpleScriptedTestProtocol.java

```diff
@@ -0,0 +1,11 @@
+package org.apache.james.mpt.script;
+
+import org.apache.james.mpt.api.HostSystem;
+
+public class SimpleScriptedTestProtocol extends GenericSimpleScriptedTestProtocol<HostSystem, SimpleScriptedTestProtocol> {
```

Comment:
It's a simple type alias but avoid hundreds of generics parameters everywhere in the code.
Believe me : it worths it !